src/main/webapp/WEB-INF/tags/shared/feedbackSubmissionEdit/response.tag

```diff
@@ -45,6 +51,31 @@
       </c:choose>:
   </div>
   <c:choose>
+    <c:when test="${questionWithResponses.feedbackParticipantCommentsOnResponsesAllowed}">
+      <button type="button" class="btn btn-default btn-xs icon-button pull-right show-frc-${addOrEdit}-form"
```

Comment:
Also, if hasCommentOnResponse, we should not show the add comment button?

Reply:
Or hide it, show it when the existing comment gets deleted.

Reply:
Currently what happens is when there is a comment, clicking add button opens edit form. And when the comment is deleted, clicking it opens add form. I think comment button should be shown at all times as it looks consistent. What do you think?

Reply:
Why not put the button together with the adding form?

So adding form consists of an adding button and a text box input.

So editing form consists of an editing button, a row display, and a text box input.

At anything only one form is visible and therefore we can alway have the button displayed.

Reply:
@xpdavid I have attached gif of current behaviour. Are you suggesting to hide blue comment button when comment is already added?

![ezgif com-video-to-gif 3](https://user-images.githubusercontent.com/24381442/42871724-b0c8335a-8a98-11e8-996f-2b9746d8f8c5.gif)

Reply:
Yes.

In addition, I think the button may not be intuitive enough for student to figure out what it is.

For adding new comment, maybe we shall have a button with text `Add Comment`. (Can just below the submission form, hide it when the adding form is shown)

For editing comment, I think we are good as the comment row already indicates that.

@damithc Any suggestions?

Reply:
I agree. As this is not a 'usual' feature that people expect to see, we may have to make it more noticeable. We can have a small button with the icon and `Add comment` text and a bit more explanatory tool tip.